src/java/org/apache/cassandra/cql3/Operator.java

```diff
@@ -58,6 +100,38 @@ public boolean isSatisfiedBy(AbstractType<?> type, ByteBuffer leftOperand, ByteB
         {
             return type.compareForCQL(leftOperand, rightOperand) < 0;
         }
+
+        @Override
+        public boolean requiresFilteringOrIndexingFor(ColumnMetadata.Kind columnKind)
+        {
+            return columnKind != ColumnMetadata.Kind.CLUSTERING;
```

Comment:
Similar to the above comment. Can this be the second clustering after the first is unrestricted?

Reply:
Same answer as previously.